todo_app/test/test_appUiTest.py

```diff
@@ -0,0 +1,42 @@
+import os
+from threading import Thread
+import pytest
+import requests
+from selenium import webdriver
+from todo_app.app import create_app
+
+KEY=os.getenv("KEY")
+TOKEN=os.getenv("TOKEN")
```

Comment:
You don't explicitly load these environment variables, and unlike Flask, pytest will not load `.env` variables by itself. Note that it will work inside VSCode as that automatically loads your `.env` file.

You can explicitly load the `.env` file in the fixture below in a similar way to how you load `.env.test` in your integration test.

Reply:
Done